### ISSUE TITLE
ino-select: make component accessible

### ISSUE THREAD

**Issue by Sl1nd** [inovex/elements#1040]
### Feature Request

Under the hodd no native select input is used. Due to the custom implementation of `ino-select`, the native accessible behavior has to be provided.

Missing parts are:

* Proper aria-roles & aria-attributes
* keyboard navigation
* Label could be connected to input (currently label is orphaned)
* [...]

Refer to [combobox pattern](https://www.w3.org/WAI/ARIA/apg/patterns/combobox/) as well as to [native select](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select?retiredLocale=de) for further information
